plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -231,6 +235,15 @@ fu_plugin_thunderbolt_remove (FuPlugin *plugin, GUdevDevice *device)
 	FuDevice *dev;
 	g_autofree gchar *id = NULL;
 
+#ifdef HAVE_DELL
+        /* On Dell systems the DELL plugin will force power */
+	/* ignore host controllers, they just go to low power mode */
```

Comment:
I think it would be good to just combine the two comments and make one and maybe explain a tiny bit more what is going on, i.e. that it will force the power in the prepare_for_update stage that. I was a tiny bit confused how it works. Also a comment around line 178 (now 182), where we ignore duplicated devices, would be good, where we indicate that this might happen on Dell  systems.